### PR TITLE
CHORES: Prepare 3.3

### PR DESCRIPTION
**Pull request by maxulysse**

### CODE REVIEW

File: .nf-core.yml [hunk 1]
```diff
@@ -2,14 +2,12 @@ repository_type: pipeline
 lint:
   files_exist:
     - conf/modules.config
-    - conf/test.config
```
Comment (asp8200) [inline .nf-core.yml:5]: Can you just remind me what this is about?

Comment (maxulysse) [reply .nf-core.yml:5]: I restored this file, so we no longer need to ignore that in the linting

File: nextflow.config [hunk 2]
```diff
@@ -311,13 +311,6 @@ env {
 // Capture exit codes from upstream processes when piping
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
-// Set default registry for Docker, Singularity and Podman independent of -profile
-// Will not be used unless Docker, Singularity and Podman are enabled
-// Set to your registry if you have a mirror of containers
-docker.registry      = 'quay.io'
```
Comment (asp8200) [inline nextflow.config:317]: Why are those three lines no longer needed?

Comment (maxulysse) [reply nextflow.config:317]: they are needed, but these are duplicates

Comment (maxulysse) [reply nextflow.config:317]: cf L280...

**Review by asp8200 — APPROVED** [review]

**Pull request merged**